src/ParcelRegistry.Api.Oslo/Convertors/ParcelStatus.cs

```diff
@@ -0,0 +1,23 @@
+namespace ParcelRegistry.Api.Oslo.Convertors
+{
+    using Be.Vlaanderen.Basisregisters.GrAr.Legacy.Perceel;
+
+    public static class ParcelStatusExtensions
+    {
+        public static PerceelStatus? MapToPerceelStatusSyndication(this ParcelStatus? status)
```

Comment:
Can you check if all methods are still used?

Reply:
isn't used, removed it